src/tests/unit/tests/DetailsView/components/save-assessment-dialog.test.tsx

```diff
@@ -66,30 +66,32 @@ describe('SaveAssessmentDialog', () => {
 });
 
 describe('dialog interaction', () => {
+    let wrapper: RenderResult;
+
     beforeEach(() => {
+        useOriginalReactElements('@fluentui/react', [
+            'Dialog',
+            'DialogFooter',
+            'Stack',
+            'Checkbox',
+        ]);
         propsStub.isOpen = true;
-        render(<SaveAssessmentDialog {...propsStub} />);
+        propsStub.userConfigurationStoreData.showSaveAssessmentDialog = false;
+        wrapper = render(<SaveAssessmentDialog {...propsStub} />);
         const gotItButtonProps = getMockComponentClassPropsForCall(PrimaryButton);
         gotItButtonProps.onClick();
```

Comment:
These lines are only needed for the first test. they should happen there.

Reply:
Hi @madalynrose made changes, please review